src/components/plan/PlanForm.svelte

```diff
@@ -152,50 +151,22 @@
   }
 
   async function onExportPlan() {
-    if (plan) {
-      if (planExportAbortController) {
-        planExportAbortController.abort();
-      }
-
-      planExportAbortController = new AbortController();
-
-      if (planExportAbortController && !planExportAbortController.signal.aborted) {
-        await exportPlan(
-          plan,
-          user,
-          (progress: number) => {
-            planExportProgress = progress;
-          },
-          Object.values(activityDirectivesMap),
-          planExportAbortController.signal,
-        );
-      }
-      planExportProgress = null;
+    if (plan && !planExporting) {
+      planExporting = true;
+      await exportPlan(plan, user, Object.values(activityDirectivesMap));
+      planExporting = false;
     }
   }
-
-  function onCancelExportPlan() {
-    planExportAbortController?.abort();
-    planExportAbortController = null;
-    planExportProgress = null;
-  }
 </script>
 
 <div class="plan-form">
   {#if plan}
     <fieldset>
       <Collapse title="Details">
         <svelte:fragment slot="right">
-          {#if planExportProgress !== null}
-            <button
-              class="st-button icon cancel-button"
-              on:click|stopPropagation={onCancelExportPlan}
-              use:tooltip={{
-                content: 'Cancel Plan Export',
-                placement: 'top',
-              }}
-            >
-              <CancellableProgressRadial progress={planExportProgress} />
+          {#if planExporting}
+            <button class="st-button icon progress" on:click|stopPropagation={() => {}}>
```

Comment:
Would it be better to just remove the button altogether since it no longer can be interacted with to cancel?

Reply:
Yes but I'm actually using the button here as a way to prevent propagation of the click event so that the collapse doesn't close. Could let it close when the export is in progress though was trying to keep it consistent with the stopPropagation on the export button.